### ISSUE TITLE
Create a bracket visualizer

### ISSUE THREAD

**Issue by Daniel17sep** [russleyshaw/Colisee#9]
Create simple (but nice looking) bracket visualizer that can display the results of the tournament.

- Should be able to show gamelogs (via link to the visualizer w/ the gamelog url is fine)
- Should not show results until a button is clicked (this is used as a simulation as if results are not known until gamelog is visualized.)
- Should be able to stream results as games are played (traditionally we have run all games before displaying the results, but this takes a long time; we need to view the results as the tournament is played.)

**Comment by russleyshaw** [comment]
@Daniel17sep Do you have an example of the data that is sent to the bracket visualizer? I should be able to get 1 or 2 other devs working on this already.

**Comment by Daniel17sep** [comment]
Yeah I can get an example, I'll try to do that soon.

**Comment by Daniel17sep** [comment]
[Example Bracket Vis Input.txt](https://github.com/russleyshaw/Colisee/files/448614/Example.Bracket.Vis.Input.txt)
This is an example of the input for the current bracket visualizer.

**Comment by Daniel17sep** [comment]
"271": {
  "player_2": "javascript2",
  "winner": "javascript2",
  "log_location": " ",
  "previous_matches": [
   270,
   267
  ],
  "player_1": "javascript1"
 },

271: Match
Player 2: Player 2
Winner: Winner of match
Log_location: Gamelog URL
Previous_matches: The matches that preceded this match (where each player came from)
Player 1: Player 1

**Comment by russleyshaw** [comment]
Current tasks

1) Make a simple API for the head server that serves a bracketvis.json
Maybe something like `GET /api/v2/bvis/` to get the most "recent" bracket visualization

2) Make a web page `GET /bvis/` that will get the bvis API and load the json file.

**Comment by ryanloeffelman** [comment]
@Daniel17sep we have a few questions for you about the sample data you provided us. We were attempting to draw the bracket on paper given the text file you provided above, and we were running into issues. Is that a real example, or is that just a sample test case? Will you be able to come to our meeting on Thursday at 2, to meet with us and we can go over this in person.

**Comment by Daniel17sep** [comment]
Yes that is the bracket from an actual tournament. I will be able to attend your meeting on Thursday.

**Comment by Daniel17sep** [comment]
Here is the bracket drawn out. I will explain on Thursday.
![tournament bracket](https://cloud.githubusercontent.com/assets/8919996/18291547/e2e8f0f8-744d-11e6-9cc0-f599a606ff84.jpg)

**Comment by russleyshaw** [comment]
assuming it stays up on the board

**Comment by ryanloeffelman** [comment]
Not really sure why there is no data showing from the JSON file, Any helpful hints would be appreciated

**Comment by russleyshaw** [comment]
Made a line comment. Here are some things to remember

- `res.sendFile(...)` takes a local filepath
- You need to have the head server running

**Comment by ryanloeffelman** [comment]
I do have the head server running, in fact, I have all the servers running just in case. When I go to localhost:3000/bracket/testData/ it does return the entire JSON file
